1-Draft/RFCNNNN-Make-Terminating-Errors-Terminate-The-Right-Way.md

```diff
@@ -0,0 +1,176 @@
+---
+RFC: RFCnnnn
+Author: Kirk Munro
+Status: Draft
+SupercededBy: 
+Version: 1.0
+Area: Engine
+Comments Due: July 15, 2019
+Plan to implement: Yes
+---
+
+# Make terminating errors terminate the right way in PowerShell
+
+By default in PowerShell, terminating errors do not actually terminate. For example, if you invoke this command in global scope, you will see the output "Why?" after the terminating error caused by the previous command:
+
+```PowerShell
+& {
+    1/0
```

Comment:
An even better (more likely encountered in the wild) scenario is using .NET methods that throw e.g.:
```
& {
    "".Substring(99)
    "Why?"
}
```
Most of the time I want this to be a terminating error.

Reply:
I like that example better. Updating my doc. Thanks.